### PR TITLE
[jp] fix Equinix Metal url 404

### PR DESCRIPTION
**Pull request by jiecloud**
fix url 404
https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/metal
to
https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/equinix

under this page
https://kubernetes.io/ja/docs/setup/production-environment/tools/kubespray/

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
@jiecloud Thanks for your contribution. I see you have already raised PR across other languages to fix this issue (#40318 #40317). Could you please revert the changes from other languages (**_en & zh-cn_**) here so that we will have a single PR per language.

**Comment by jiecloud** [comment]
> 

The other two language PRs have been merged by other developers. If you think this PR is feasible, please merge this language version as well. I'm not very familiar with the rules yet, I will learn more later. Thank you.
@dipesh-rawat

File: content/en/docs/setup/production-environment/tools/kubespray.md
Comment (dipesh-rawat) [inline content/en/docs/setup/production-environment/tools/kubespray.md:1]: @jiecloud Could you please omit this change from the Japanese PR.

File: content/zh-cn/docs/setup/production-environment/tools/kubespray.md
Comment (dipesh-rawat) [inline content/zh-cn/docs/setup/production-environment/tools/kubespray.md:1]: @jiecloud Also, could you please omit this change from the Japanese PR.

**Comment by jiecloud** [comment]
ok , I will close this pr, thanks. @dipesh-rawat

**Pull request closed without merging**